src/pyramid/authentication.py

```diff
@@ -1035,8 +1035,8 @@ def _get_cookies(self, request, value, max_age=None):
                 domains.append('.' + cur_domain.split('.', 1)[1])
             else:
                 domains.append(None)
-                domains.append(cur_domain)
                 if self.wild_domain:
+                    domains.append(cur_domain)
                     domains.append('.' + cur_domain)
```

Comment:
We should drop this line of the code, it is unnecessary and we should probably change the logic to either append `cur_domain` or append `None`, not do both. There is no reason to send two cookies in this case.